File: Data/Models/GeneroUsuario.cs
namespace TrabalhoAPI.Data.Models
{
    public class GeneroUsuario
    {
        public int Id { get; set; }
        public int UsuarioId { get; set; }
        public Usuario? Usuario { get; set; }
        public int GeneroId { get; set; }
        public Genero? Genero { get; set; }
    }
}

File: src/app/GeneroUsuario.ts
import { Genero } from "./Genero";
import { Usuario } from "./Usuario";

export class GeneroUsuario{
    id: number = 0;
    usuarioId: number = 0;
    usuario: Usuario | null = null;
    generoId: number = 0;
    genero: Genero | null = null;
}